api/topn_agg.md

```diff
@@ -0,0 +1,71 @@
+# topn_agg() <tag type="toolkit" content="Toolkit" /><tag type="experimental" content="Experimental" />
+Produces an aggregate that can be passed to the [`topn` function][topn] to
+calculate the `n` most-frequent values in a column.
+```sql
+topn_agg (
+    FIXME INTEGER,
+    value AnyElement
+) RETURNS TopnAggregate
```

Comment:
Guessed at the name of the return parameter by extrapolation

Reply:
It's called SpaceSavingAggregate. It's returned by both topn_agg and freq_agg.